components/Footer/Footer.tsx

```diff
@@ -0,0 +1,23 @@
+import Image from "next/image";
+import Link from "next/link";
+import type { FC } from "react";
+import Logo from "../logo";
+
+interface FooterProps {}
+
+export const Footer: FC<FooterProps> = ({}) => {
+  return (
+    <div className="fixed bottom-0 w-full flex items-baseline justify-center z-10 gap-1 rounded-lg bg-[#343434] py-3">
+      <p className="text-lg text-white ">
```

Comment:
we are not using `white` anywhere in the project. so just update it with `text-gray-200`